src/ActivityRepository.js

```diff
@@ -62,31 +62,13 @@ class ActivityRepository {
     };
   }
 
-  getAverageStairsDay(date) {
+  getAverages(date, property) {
     const filteredDate = this.getFilteredDate(date);
-    const totalStairs = filteredDate.reduce((total, log) => {
-      total += log.flightsOfStairs;
+    const total = filteredDate.reduce((total, log) => {
+      total += log[property];
       return total;
     }, 0);
-    return Math.round(totalStairs / filteredDate.length);
-  }
-
-  getAverageStepsDay(date) {
-    const filteredDate = this.getFilteredDate(date);
```

Comment:
This looks very identical to the methods I had, I was going to go this exact route but took a different approach, I ended up creating a helper method like so:
![Helper method](https://user-images.githubusercontent.com/51575613/67167103-2e87d600-f353-11e9-8a6d-28aa84e2a997.jpeg)
In this method, I extracted the reduce function and made it dynamic so that when invoking it in the other methods, it's already doing the logic. I found this very helpful because this saved me from having to overlook all the files that call on these methods and adjusting them to what I wrote.

Reply:
This difference in style/approach is just fine - you might find that your particular solution ends up being a good use-case for inheritance (It also may not, but just something to think about. I'd have to look into it further to see if that ends up being true.)  You are still leveraging args/params to make your helper method more dynamic, which adds some consistency to how you're solving these problems. If the inconsistency in approach is something that ends up really bothering you, you can all make a decision as a team on how you'd like to refactor.